modules/andes-core/broker/src/main/java/org/wso2/andes/store/cassandra/HectorBasedMessageStoreImpl.java

```diff
@@ -89,6 +88,11 @@
     private long lastRecoveryMessageId;
 
     /**
+     * The comma is used to append message IDs to one another when performing column operations
+     */
+    private final String comma = " , ";
```

Comment:
Is this is a constant, it should be static. Also the variable name should be uppercase.